client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -189,9 +189,7 @@ const HostingFeatures = () => {
 	let buttons;
 	if ( isTransferInProgress && config.isEnabled( 'hosting-overview-refinements' ) ) {
 		title = translate( 'Activating hosting features' );
-		description = translate(
-			'Stay tuned, as we activate the following features included in your plan'
-		);
+		description = translate( 'This place will refresh once the activation is completed.' );
```

Comment:
```suggestion
		description = translate( 'This section will refresh once the activation is completed.' );
```
I know this phrasing comes from the design spec, but the word 'place' feels off to me here. 'Section' feels like a better fit 🤔

We could try something like "Your new hosting features will appear here automatically when they're ready!"

cc @nuriapenya - I don't want to derail the copy of the original design on you, but what do you think?

We may also want to consider adding a translation comment, to avoid any similar ambiguity around word choice for other languages.

Reply:
Oh, apologies for it! It was meant to read "Page," not place 🤦🏽‍♀️ 

But I like your suggestion. I would only change one small thing:

> "**_The_** hosting features will appear here automatically when they're ready!"

We avoid saying "Your" as much as we can, because many times users are building for other people. Is not a hard rule, but something we are a bit mindful for.

Thank you for checking!! 🙏🏽

Reply:
Nice, that sounds good to me 🙂 

> We avoid saying "Your" as much as we can, because many times users are building for other people. Is not a hard rule, but something we are a bit mindful for. 

Oh cool. That makes a lot of sense, thanks for sharing it!